Disable manpages for coreutils. These are generated by running the host version --help
which is going to give incorrect results so its better simply not to do this at all.

RP 2017/3/9
Upstream-Status: Inappropriate [OE cross compile specific hack]

Index: coreutils-6.9/Makefile.am
===================================================================
--- coreutils-6.9.orig/Makefile.am
+++ coreutils-6.9/Makefile.am
@@ -17,7 +17,7 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
 # 02110-1301, USA.
 
-SUBDIRS = lib src doc man po tests
+SUBDIRS = lib src doc po tests
 EXTRA_DIST = Makefile.cfg Makefile.maint GNUmakefile \
   .gitignore \
   .hgignore \
